students/pkoleyni/Lesson03/list_lab.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+fruit_list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+def add_fruit(seq):
+    seq.append(input('Enter name of a fruit >'))
+    return seq
+
+def find_by_index(seq):
+    user_input = input('Enter a number in range of 1 to {} >'.format(len(seq) + 1))
+    return ('You have entered {} which is {} from the fruit_list'.format(user_input,seq[int(user_input)-1]) )
+
+def find_by_letter(seq, letter ='P'):
+    for item in fruit_list:
+        if letter in item:
```

Comment:
your logic should ensure that it checks that the string starts with P, not that P is somewhere in the word.